main.cpp

```diff
@@ -132,39 +190,13 @@ int main(int argc, char *argv[])
     const float x = static_cast<float>(std::atof(argv[1]));
     const float y = static_cast<float>(std::atof(argv[2]));
     const std::string file_name(argv[3]);
-    render_scene(display.data(), width, height, {
-            { 0.0f, 0.0f, -200.0f },       // eye
-            {                              // spheres
-                { { x, y, 200.0f }, 100.0f },
-                { { x + 100.0f, y, 100.0f }, 100.0f }
-            },
-            {                              // walls
-                {
-                    { 0, 0, -1, 500 },    // p
-                    { 1.0f, 1.0f, 1.0f }     // c
-                },
-                {
-                    { 0, 0, 1, 500 },     // p
-                    { 1.0f, 0, 0 }         // c
-                },
-                {
-                    { 1, 0, 0, 500 },    // p
-                    { 0, 1.0f, 0 }        // c
-                },
-                {
-                    { -1, 0, 0, 500 },    // p
-                    { 0, 0, 1.0f }        // c
-                },
-                {
-                    { 0, 1, 0, 500 },    // p
-                    { 1.0f, 1.0f, 0 }        // c
-                },
-                {
-                    { 0, -1, 0, 500 },    // p
-                    { 1.0f, 0, 1.0f }        // c
-                }
-            }
-        });
+
+    auto scene = load_scene_from_file("./scene.txt");
+    scene.spheres.push_back({ { x, y, 200.0f }, 100.0f });
+    scene.spheres.push_back({ { x + 100.0f, y, 100.0f }, 100.0f } );
```

Comment:
What's the point of this hardcode? Please move these spheres to the scene.txt file.

Reply:
Oh, they depend on a CLI parameter right. Sorry. :) Forget about this one. I'll take care ot it.